dev-util/umockdev/umockdev-0.10.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
+
+inherit python-any-r1
+
+MY_PV=${PV%??}
```

Comment:
Why are you naming it `0.10.0` if upstream calls it `0.10` everywhere?

Reply:
Not everywhere, .../trunk/0.10.0/..., I can do it the other way around and add .0 to PV or hardcode that line.